src/components/Alert/styles/Border.js

```diff
@@ -19,17 +30,10 @@ const AlertBorder = styled.div`
   animation: ${fadeIn} 0.2s ease;
   transition: opacity 0.2s ease;
 
-  background: ${({ theme, type }) => {
-    switch (type) {
-      case 'success':
-        return get('colors.positive.light');
-      case 'error':
-        return get('colors.negative.light');
-      default:
-        return get('colors.primary.light');
-    }
-  }};
-
+  background: ${getBgColorForType};
+  
+  & a::before { background-color: ${getBgColorForType} !important;}
```

Comment:
Is `!important` needed here? Can you just make this selector more specific than the default color selector ([link](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity))? For instance, you can add additional `&` to increase specificity sometimes. You can also import `TextLink` here and interpolate it to access its class name: `& ${TextLink}::before { ... }`.